.github/workflows/release.yaml

```
@@ -0,0 +1,57 @@
+        run: make rpm
+      - name: Get Artifacts
+        id: get_artifacts
+        run: |
+          echo "::set-output name=srcpath::$(ls ./dist/src/*.tar.gz)"
+          echo "::set-output name=srcfilename::$(basename ./dist/src/*.tar.gz)"
+          echo "::set-output name=binpath::$(ls ./dist/bin/tapir-cli)"
+          echo "::set-output name=binfilename::$(basename ./dist/bin/tapir-cli)"
+          echo "::set-output name=rpmpath::$(ls ./dist/rpm/RPMS/x86_64/*rpm)"
+          echo "::set-output name=rpmfilename::$(basename ./dist/rpm/RPMS/x86_64/*rpm)"
+      - name: Upload Tarball
+        run: ""
```

actionlint (1.7.4)

string should not be empty

(syntax-check)